libass/ass.h

```diff
@@ -30,6 +30,21 @@
 extern "C" {
 #endif
 
+#if (defined(__GNUC__) && ((__GNUC__ > 3) || (__GNUC__ == 3 && __GNUC_MINOR__ >= 1))) || defined(__clang__)
+    #define ASS_DEPRECATE(msg)  __attribute__((deprecated(msg)))
+    #if __GNUC__ > 5 || defined(__clang__)
+        #define ASS_DEPRECATE_ENUM(msg) __attribute__((deprecated(msg)))
+    #else
+        #define ASS_DEPRECATE_ENUM(msg)
+    #endif
+#elif defined(_MSC_VER)
+    #define ASS_DEPRECATE(msg)  __declspec(deprecated)
```

Comment:
MSVC apparently supports passing a message here (`__declspec(deprecated(msg))`), so if we're going to keep this I think we should include that. Also, another double-space.

Reply:
Thanks, I missed that.